addons/website_event/models/event.py

```diff
@@ -41,6 +46,12 @@ def _compute_is_participating(self):
         else:
             self.is_participating = False
 
+    def is_one_day(self):
```

Comment:
Never sure if we should make this a computed field or not (especially since you use it in views)... to check with JKE.

Reply:
@JKE-be